---
title: Weekly Progress Report
author:
- name: Camille Scott
date: 2015-04-06
...

## Lamprey

### Finished

* 2013-lamprey repository was getting unwieldy and messy. A new repo has been created, at https://github.com/camillescott/2015-petMarSB
* In the new repo, I have mostly unified the reproducible pipeline. The main driver script is just `pipeline`, at the top level.
    - Usage: 

        ./pipeline --resources-metadata <resources.json> --config-metadata <config.json> 
        --local-file-dir <directory-with-local-data> [run|list|clean|etc] ... <pydoit args>

    - All metadata is now being stored in JSON format. For example, the metadata for a sample is stored like:

        "Gill": {
        "access": "local_file",
        "condition": "",
        "db_type": "nucl",
        "filename": "Gill.fq.gz",
        "flowcell": "42UL4",
        "label": "gill_42UL4",
        "meta_type": "sample",
        "paired": false,
        "phred": "phred33",
        "strand": "+/-",
        "tissue": "gill"
        }
    
    - Now has an explicit license -- CC-BY (CC Attribution), (Is this an appropriate license to use?). 
    

### TODO next

* Fix up notebooks in new repo
* Use new abundance info to build gene models from existing transcripts
* Get detailed comparison metrics between existing gene models, transcript alignments, and new transcript-based gene models

## Other

* Continued work with Greg Wilson on the Teaching Tech Together workshop
* Working with Alita to reboot our Avida genetic modularity project and push it out the door before I move
